orchestra/src/lib.rs

```diff
@@ -512,6 +515,15 @@ where
 		msg: OutgoingMessage,
 	) -> Result<(), metered::TrySendError<OutgoingMessage>>;
 
+	/// Tries to send a direct priority message to some other `Subsystem`, routed based on message type.
+	/// This method is useful for cases where the message queue is bounded and the message is ok
+	/// to be dropped if the queue is full. If the queue is full, this method will return an error.
+	/// This method is not async and will not block the current task.
```

Comment:
It doesn't make much sense to drop priority messages.

Reply:
Make sense. Does it mean that we should not have the try_priority_send at all?